importer/tag/src/main/java/org/itsallcode/openfasttrace/importer/tag/LongTagImportingLineConsumer.java

```diff
@@ -21,13 +20,17 @@ class LongTagImportingLineConsumer extends RegexLineConsumer
             .getLogger(LongTagImportingLineConsumer.class.getName());
 
     private static final String COVERING_ARTIFACT_TYPE_PATTERN = "\\p{Alpha}+";
+    // [impl->dsn~import.full-coverage-tag-with-revision~1]
+    private static final String COVERING_TAG = "(" + COVERING_ARTIFACT_TYPE_PATTERN + ")"
+            + optional(SpecificationItemId.ARTIFACT_TYPE_SEPARATOR + SpecificationItemId.REVISION_SEPARATOR
+                    + SpecificationItemId.ITEM_REVISION_PATTERN);
     private static final String OPTIONAL_WHITESPACE = "\\s*";
     private static final String TAG_PREFIX = "\\[";
     private static final String TAG_SUFFIX = "\\]";
     private static final String NEEDS_COVERAGE = ">>" + OPTIONAL_WHITESPACE + "(\\p{Alpha}+(?:" + OPTIONAL_WHITESPACE
             + "," + OPTIONAL_WHITESPACE + "\\p{Alpha}+)*)";
     private static final String TAG_REGEX = TAG_PREFIX + OPTIONAL_WHITESPACE//
-            + "(" + COVERING_ARTIFACT_TYPE_PATTERN + ")" //
+            + COVERING_TAG //
```

Comment:
This is a deviation from the scheme of the other regex elements. In all others we make the grouping brackets transparent (intended) here we include it into the covering tag fragement. I think you should revert this.